http/sys_rekey.go

```diff
@@ -231,23 +235,161 @@ func handleSysRekeyUpdate(core *vault.Core, recovery bool) http.Handler {
 	})
 }
 
+func handleSysRekeyVerify(core *vault.Core, recovery bool) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		standby, _ := core.Standby()
```

Comment:
My guess is that the person who wrote that method thought they were future proofing it by including an error in the signature, so that in the future if there was an error case... :-)

I don't have strong feelings. @briankassouf @chrishoffman ?

Reply:
We might as well remove the error from that function signature. Doesn't seem likely it'll get used since standby is set out of band.